src/openfermion/transforms/_unitary_rotations.py

```diff
@@ -0,0 +1,33 @@
+#   Licensed under the Apache License, Version 2.0 (the "License");
+#   you may not use this file except in compliance with the License.
+#   You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+
+"""Functions for unitary transformations of operators"""
+import numpy
+
+
+def rotate_qubit_by_pauli(qop, pauli, angle):
+    '''
```

Comment:
Please add the "args" and "returns" doc strings. I am not sure what type of object "pauli" is.

Reply:
Done